Microsoft.Toolkit.Uwp.UI/Extensions/Mouse/Mouse.cs

```diff
@@ -0,0 +1,114 @@
+﻿// ******************************************************************
+// Copyright (c) Microsoft. All rights reserved.
+// This code is licensed under the MIT License (MIT).
+// THE CODE IS PROVIDED “AS IS”, WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED,
+// INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.
+// IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM,
+// DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT,
+// TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH
+// THE CODE OR THE USE OR OTHER DEALINGS IN THE CODE.
+// ******************************************************************
+
+using System;
+using System.Collections.Generic;
+using Windows.UI.Core;
+using Windows.UI.Xaml;
+using Windows.UI.Xaml.Input;
+
+namespace Microsoft.Toolkit.Uwp.UI.Extensions
+{
+   /// <summary>
+   /// Helper class for easily changing the mouseover cursor type.
+   /// </summary>
+   public class Mouse : DependencyObject
+   {
+      /// <summary>
+      /// Dependency property for specifying the target <see cref="CoreCursorType"/> to be shown
+      /// over the target <see cref="FrameworkElement"/>.
+      /// </summary>
+      public static readonly DependencyProperty CursorProperty =
+          DependencyProperty.RegisterAttached("Cursor", typeof(CoreCursorType), typeof(Mouse), new PropertyMetadata(CoreCursorType.Arrow, CursorChanged));
+
+      /// <summary>
+      /// Set the target <see cref="CoreCursorType"/>.
+      /// </summary>
+      /// <param name="element">Object where the selector cursor type should be shown.</param>
+      /// <param name="value">Target cursor type value.</param>
+      public static void SetCursor(FrameworkElement element, CoreCursorType value)
```

Comment:
Can we use DependencyObject instead of FrameworkElement ? This would allow using x:Bind in addition to Binding should this be driven through code.

Reply:
You should always limit attached properties to the elements that they apply to. If there is a bug with x:Bind that doesn't change this rule.

Reply:
FrameworkElement is the lowest common class implementing both PointerEntered, PointerExited and Unloaded events, it cannot be changed to DependencyObject.

Reply:
@skendrot maybe you should talk to the xaml team. We can't not support x:Bind. The dependency object can then be cast to needed type.

Reply:
@hermitdave Just tested and the CoreCursor value can be set with x:Bind. FrameworkElement inherits from DependencyObject. I don't see any problem here with the use of FrameworkElement as the lowest common class.